jxl/src/frame.rs

```diff
@@ -209,18 +209,26 @@ mod test_frame {
 
     use super::Frame;
 
-    fn read_frame(image: &[u8]) -> Result<Frame, Error> {
+    fn read_frames(image: &[u8]) -> Result<Vec<Frame>, Error> {
```

Comment:
I just noticed now that the mod is called test_frame... why is it not just test?

Reply:
done